install_config/topics/ocp_osp_provisioning.adoc

```diff
@@ -0,0 +1,273 @@
+[[inventory_provision]]
+=== Preparing the Inventory for Provisioning
+
+With the installation of the `openshift-ansible` package complete via our
+previous steps, there resides a 
+`sample-inventory` directory that we will copy to our `cloud-user` home directory
+of the deployment host.
+
+On the deployment host, 
+
+----
+$ cp -r /usr/share/ansible/openshift-ansible/playbooks/openstack/sample-inventory/ ~/inventory
+----
+
+Within this inventory directory, the _all.yml_ file contains all the different 
+parameters that must be set in to order to achieve successful provisioning of 
+the RHOCP instances. The _OSEv3.yml_ file contains some references required by
+the _all.yml_ file plus all the different OpenShift cluster parameter installation
+customization one would want. 
+
+[[all.yml]]
+==== all.yml
+
+The _all.yml_ has many options that can be modified to meet your specific needs.
+The information gathered in this file is for the provisioning portion of the instances
+required for a successful deployment of {product-title}. It
+is important to review these carefully. This document will provide a condensed 
+version of the _all.yml_ and focus on the most critical parameters that need to 
+be set for a successful deployment.
+
+
+[subs=+quotes]
+----
+*$ cat ~/inventory/group_vars/all.yml*
+---
+openshift_openstack_clusterid: "openshift"
+openshift_openstack_public_dns_domain: *"example.com"*
+openshift_openstack_dns_nameservers: *["10.19.115.228"]*
+openshift_openstack_public_hostname_suffix: "-public"
```

Comment:
openshift_openstack_public_hostname_suffix doesn’t exist any longer in “inventory/group_vars/all.yml"

Reply:
They are definitely both required. I'd say this is a bug with the ShiftStack DFG team. Will open a BZ to have them backport this to OCP3.10

Reply:
Opened a BZ: https://bugzilla.redhat.com/show_bug.cgi?id=1631853